.github/workflows/python-conda-build.yaml

```diff
@@ -36,4 +36,11 @@ jobs:
           export PATH="$GITHUB_PATH:$PATH"
           source activate scAR
           conda install pytest
-          pytest scAR
\ No newline at end of file
+          pytest scAR
+      - name: Run integration tests
+        run: |
+          export PATH="$GITHUB_PATH:$PATH"
+          source activate scAR
+          cd scAR/test
+          python test_scar_integration.py
+          cd -
```

Comment:
I think the issue was that the integration test was recognised as a unit test, because the script was starting with `test_`. Let's see if the issue will be solved now...

Reply:
At the end I had to move it to a different directory and ignore the directory containing the integration test. I will wait for @mr-nvs to have a look as he is more experienced with pytest.